src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/hosts.component.ts

```diff
@@ -184,6 +186,37 @@ export class HostsComponent extends ListWithDetails implements OnInit {
         isHidden: this.clusterCreation,
         flexGrow: 1,
         pipe: this.cephShortVersionPipe
+      },
+      {
+        name: $localize`IP Address`,
+        prop: 'addr',
+        flexGrow: 1
+      },
+      {
+        name: $localize`CPUs`,
+        prop: 'cpu_count',
+        flexGrow: 1
+      },
+      {
+        name: $localize`Memory`,
+        prop: 'memory_available_kb',
+        pipe: this.dimlessPipe,
+        flexGrow: 1
+      },
+      {
+        name: $localize`Raw Capacity`,
+        prop: 'hdd_capacity',
+        flexGrow: 1
+      },
+      {
+        name: $localize`Disk Types`,
```

Comment:
As Ernesto said, I still think this should be `Disk Count`, `Disk Number` or something like that. With `Disk Types` I would expect a list of types like the labels column. What do you think @avanthakkar?

Reply:
> As Ernesto said, I still think this should be `Disk Count`, `Disk Number` or something like that. With `Disk Types` I would expect a list of types like the labels column. What do you think @avanthakkar?

Yes there's disk types two..if u see the screenshot attached in PR description it's not just count, it says `4 HDD` so it also represents type of disk.

Reply:
I meant that with `types` I wouldn't expect to see a number but with `number` seeing `hdd` is ok. Probably I'm being super picky, `Disk Types` works fine too.

Reply:
I changed it to `Disks` now to cover both types and count..what u think? @pereman2